### ISSUE TITLE
Accessibility Testing - Colour Contrasts

### ISSUE THREAD

**Issue by patricia568** [Jowens96/animal-rescue-website#6]
Hello!

checkmycolours.com says that there doesn't seem to be enough colour contrast between some of your type and the colour backgrounds (mostly the white text on bright blue background, as well as the black text on bright blue background). Buttons are good to go, though!

Good job, it looks great! :)

- Patricia

**Comment by Jowens96** [comment]
I've tried to adjust the colours of elements like the background and skip links, but to no avail. They aren't changing.
